tests/pages/firefox/base.py

```diff
@@ -24,10 +29,16 @@ class FamilyNavigation(PageRegion):
         _nav_menu_locator = (By.ID, 'fxfamilynav-tertiarynav')
 
         def open_menu(self):
-            self.root.find_element(*self._nav_button_locator).click()
-            Wait(self.selenium, self.timeout).until(
-                expected.visibility_of_element_located(self._nav_menu_locator))
+            self.find_element(self._nav_button_locator).click()
+            self.wait.until(expected.visibility_of_element_located(self._nav_menu_locator))
 
         @property
-        def is_displayed(self):
+        def is_menu_displayed(self):
             return self.is_element_displayed(self._nav_menu_locator)
+
+
+class FirefoxBasePageRegion(PageRegion):
+
+    def scroll_element_into_view(self, locator):
+        return super(FirefoxBasePageRegion, self).scroll_element_into_view(
```

Comment:
It's too bad we have to repeat this. I think I understand why it's required, but I wonder if there's another solution? Maybe something like a mixin?